docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,70 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+{es} supports two security models for cross-cluster search (CCS):
+
+* <<esql-ccs-security-model-certificate, TLS certificate authentication>>
+* <<esql-ccs-security-model-api-key, API key authentication>>
+
+[discrete]
+[[esql-ccs-security-model-certificate]]
+===== TLS certificate authentication
+
+TLS certificate authentication makes sense in single administrator scenarios where you have full control over both clusters.
+Importantly, you must ensure that named roles and their privileges are identical in both clusters.
```

Comment:
What's the reason for not keeping strong recommendation that they be strictly identical?

Reply:
"must ensure...identical" is not technically accurate.  recommending that they exist and are identical is sane approach, but it is not technically required. e.g. if you don't it will still work, but the results may be surprising. there are likely some cases where you intentionally omit some permissions across clusters.